.changeset/early-spoons-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"llamaindex": minor
```

Comment:
@yisding changed this to patch (i assume we don't want a minor release for this change)